lib/cmds/merge_cmds/export.ts

```diff
@@ -142,6 +152,10 @@ const exportEnvironmentMigration = async ({
       targetEnvironmentId,
       spaceId: activeSpaceId
     })
+
+    if (!migration) {
+      throw new Error('Migration could not be exported.')
```

Comment:
This is the same error message as above - should we put it in a const so it's reusable?